README.md

```diff
@@ -0,0 +1,117 @@
+# Cribl Search Grafana Plugin
+
+This is a Data Source Plugin for Grafana that pulls from Cribl Search.
```

Comment:
What is Cribl Search? How does it get triggered? Link to product? Samples?

Reply:
Added product link.  There are 2 screenshots included in `src/img`, but it (the plugin validator) wouldn't allow me to use relative links to those images.

I'm going to ask the Community about this, about where those screenshots end up if they're not referenced in README.

Reply:
https://community.grafana.com/t/question-about-screenshots/100172

Reply:
The screenshots end up showing up in the catalog entry.  Can't reference them from README on the plugin page, but they _will_ show up in the catalog at least.